pkg/customers/approval_test.go

```diff
@@ -113,3 +78,12 @@ func TestCustomers__containsValidPrimaryAddress(t *testing.T) {
 		t.Error("Address is Secondary")
 	}
 }
+
+func containsValidPrimaryAddress(addrs []client.CustomerAddress) bool {
```

Comment:
Did I remove calling this in `pkg/customers/approval.go` at some point?

Reply:
Not sure on that, but moved it here since it wasn't being used in `approval.go`. Let me know if you'd like me to move it back though.

Reply:
I'm pretty sure we should be verifying `if len(cust.Addresses) > 0` then exactly one has `.Type == Primary`

Reply:
Ok, I moved this back into `approval.go` but will add this check in a separate PR. 

Created related issue #215